gc/base/standard/Scavenger.hpp

```diff
@@ -61,6 +61,14 @@ struct OMR_VM;
 
 extern "C" void concurrentScavengerAsyncCallbackHandler(OMR_VMThread *omrVMThread);
 
+struct CycleTimes {
```

Comment:
Do we need this defined twice, Here and L81? Not sure why we would need to do this